Library/Homebrew/cmd/style.rb

```diff
@@ -120,7 +120,7 @@ def check_style_impl(files, output_type, options = {})
       # exitstatus can also be nil if RuboCop process crashes, e.g. due to
       # native extension problems.
       # JSON needs to be at least 2 characters.
-      if !status.success? || json.to_s.length < 2
+      if status.exitstatus > 1|| json.to_s.length < 2
```

Comment:
It's worth noting what the old code was I think: https://github.com/Homebrew/brew/pull/2690/files

Why is `if $?.exitstatus.nil? || $?.exitstatus > 1 || json.to_s.length < 2` no loner appropriate?